### ISSUE TITLE
which data set is used to generate this model D40_C3.h5?

### ISSUE THREAD

**Issue by primejava** [eeskimez/Talking-Face-Landmarks-from-Speech#5]
Hello, I would like to ask how much data you used to train the model? I want to use a five-minute video of a news anchor for training, and then use the audio extracted from the video for testing, so that a correct result can be produced (I have not considered the problem of fitting)?

**Comment by eeskimez** [comment]
Hi, we used Grid corpus (paper name: "An audio-visual corpus for speech perception and automatic speech recognition") to train our system. Please refer to our paper for the training details, you can find the link on the main page. If you are training your system with the same identity (the same person talking through the videos), you don't need to normalize the face landmarks since it is a speaker-dependent system. In that case, 5 minutes video is fine. But if you aim a speaker-independent system, you may need more data that includes various identities.

**Comment by primejava** [comment]
@eeskimez Thank you for your reply, but I tried to use an hour of Obama's video for training. I adjusted the frame rate of the video to 25FPS and clipped the video so that Obama's face occupied the majority of the picture. Then I divided the video into three seconds. But After training, model predicting identical "open-mouth" keypoints. Can you tell me what I did wrong in data processing? I look forward to your reply.

**Comment by primejava** [comment]
yes ,I don't need to normalize the face landmarks .I'm a beginner. Can you point out which code needs to be deleted?

**Comment by eeskimez** [comment]
@primejava You can comment out [this line](https://github.com/eeskimez/Talking-Face-Landmarks-from-Speech/blob/ce1e57b2e40f8292e8650c784074cd86fcaecb7f/featureExtractor.py#L109) and change next line to `frames = fnorm.unitNorm(transferredFrames)`. Also, you can generate videos of these ground truth landmarks to validate their quality before training.
